STM32F405RGTx/Core/Inc/lut.hpp

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include <stdint.h>
+
+// lookup tables
+extern float ADC_TO_TEMP_LUT[4096];
```

Comment:
This is declared here, but is never defined anywhere.

Reply:
The correct lut is now added